install-scripts/mac/skm-install-mac.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
```

Comment:
**suggestion:** Could we add a blank line after the hashbang? Additionally, adding a couple of flags to catch errors earlier.

```sh
#!/bin/bash

set -eu
```